zerver/tests/test_subs.py

```diff
@@ -1014,6 +1014,57 @@ def test_create_stream_by_waiting_period_threshold(self) -> None:
         )
         self.assert_json_success(result)
 
+    def test_invite_to_stream_by_invite_period_threshold(self) -> None:
+        """
+        Non admin users with account age greater or equal to the invite
+        to stream threshold should be able to invite others to a stream.
+        """
+        hamlet_user = self.example_user('hamlet')
+        hamlet_user.date_joined = timezone_now()
+        hamlet_user.save()
+
+        cordelia_user = self.example_user('cordelia')
+        cordelia_user.date_joined = timezone_now()
+        cordelia_user.save()
+
+        hamlet_email = hamlet_user.email
+        cordelia_email = cordelia_user.email
+
+        self.login(hamlet_email)
+        do_change_is_admin(hamlet_user, True)
+
+        # Hamlet creates a stream as an admin..
+        stream_name = ['waitingperiodtest']
+        result = self.common_subscribe_to_streams(hamlet_email, stream_name)
```

Comment:
We should check the return status on this

Reply:
Fixed, thanks!